modules/terminal/terminal/src/lib.rs

```diff
@@ -81,14 +92,39 @@ impl Guest for Component {
             };
             match message {
                 wit::Message::Request(wit::Request { body, .. }) => {
-                    if state.our != source {
+                    if state.our == source {
+                        match parse_command(
+                            &mut state,
+                            std::str::from_utf8(&body).unwrap_or_default(),
+                        ) {
+                            Ok(()) => continue,
+                            Err(e) => println!("terminal: {e}"),
+                        }
+                    } else if state.our.node == source.node {
+                        let Ok(edit_aliases) = serde_json::from_slice::<EditAliases>(&body) else {
+                            println!("terminal: invalid action!");
+                            continue;
+                        };
+
+                        match edit_aliases.process {
+                            Some(process) => {
+                                state
+                                    .aliases
+                                    .insert(edit_aliases.alias.clone(), process.clone());
+                                println!(
+                                    "terminal: alias {:?} set to {:?}",
+                                    edit_aliases.alias,
+                                    process.to_string()
```

Comment:
instead of doing `process.to_string()` you can just print it without the `:?` debug, it will format automatically. same for alias i believe, no need to debug-print